release/nightly_tests/dataset/read_and_consume_benchmark.py

```diff
@@ -78,7 +78,7 @@ def consume_fn(ds):
     elif args.iter_torch_batches:
 
         def consume_fn(ds):
-            for _ in ds.iter_torch_batches():
+            for _ in ds.iter_torch_batches(device="cuda"):
```

Comment:
Would be nice to have a block comment on what this release does does.

Reply:
Added a comment for this branch